services/tasks-svc/internal/room/room.go

```diff
@@ -193,10 +263,13 @@ func (ServiceServer) DeleteRoom(ctx context.Context, req *pb.DeleteRoomRequest)
 
 	// TODO: Handle beds
 
+	// TODO: remove from spice (also beds)
+
+	// return
```

Comment:
```suggestion
	// TODO: remove from spice (also beds)

```

Why not implementing this TODO right now?

Reply:
say we remove an org, then we have to remove the associated wards as well, then the related beds. there is no cascade delete. what im saying is that this is not a trivial thing and should be dealt with in a more focused PR as part of the https://github.com/helpwave/services/issues/877 tracking issue